doc/Custom.md

```diff
@@ -1,7 +1,7 @@
 ### Connect to an unsecured kafka cluster accessible on `0.0.0.0:9092`
 ```sh
  docker run --rm \
-  --network host \
+  -p "8080:80" \
```

Comment:
Can you also update the URL below? ("CP is available on..")

Reply:
Done